### DISCUSSION TITLE
got error vtk remote rendering using vtk as client

### DISCUSSION THREAD

**Discussion by guozhi72** [Kitware/trame#279] [General]
Hi, 
I manage to make vtk_server.py worked using paravaiew as client most codes from 
https://github.com/dmreagan/vtk-remote-render/blob/master/client/src/index.js
But when I change the client code using vtk client, client code mostly from:
https://kitware.github.io/vtk-js/examples/RemoteView.html
I got below error( in both case , I used  same vtk_server.py from kitware link show above)
![error](https://github.com/Kitware/trame/assets/84816693/ba87e790-c08b-488e-8c8b-f9558bbefb42)
My client code as below, same code structure worked if I used paraview client.
![js_code](https://github.com/Kitware/trame/assets/84816693/13f5a704-3b97-494b-88f4-82539d3743b5)
Can someone help? I can send entire code if need( but the code is embeded into anugar project, paraview as client, I make it work. VTK as client, got error.

**Comment by jourdain** [comment]
I would suggest [a support contract from Kitware](https://www.kitware.com/contact/)

**Comment by guozhi72** [comment]
It is fair to consult with kitware. But it seems this is know issue, let me to do more search. 
![image](https://github.com/Kitware/trame/assets/84816693/589bc74b-f0c1-498f-bfce-11b4e2bf0980)

**Reply by guozhi72** [reply]
but I don't really know

**Reply by jourdain** [reply]
Since August 2021, twisted is not used.

**Reply by jourdain** [reply]
And not the same error

**Comment by jourdain** [comment]
Closing since it is not trame related.
